.github/workflows/test.yml

```diff
@@ -23,11 +23,26 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 14.x
+      - name: Get cddl version
+        run: curl -s https://crates.io/api/v1/crates/cddl | python3 -c "import sys, json; print(json.load(sys.stdin)['crate']['max_stable_version'])" > cddl.txt && echo cddl.txt
```

Comment:
You can also pipe the output into `tee cddl.txt` to both write to stdut and to a file.

Reply:
Ah good to know. Will make the change which is much nicer.